pkg/processor/runtime/python/runtime.go

```diff
@@ -266,6 +265,26 @@ func (py *python) handleResponseLog(functionLogger nuclio.Logger, response []byt
 	logFunc(message, vars...)
 }
 
+func (py *python) handleReponseMetric(functionLogger nuclio.Logger, response []byte) {
+	var metrics struct {
+		Duration float64 `json:"duration"`
```

Comment:
Would be happy if this field contained the units (e.g. `DurationMs`)

Reply:
Done